contracts/vaults/base/RibbonVault.sol

```diff
@@ -574,7 +580,8 @@ contract RibbonVault is OptionsVaultStorage {
             depositReceipt.getSharesFromReceipt(
                 vaultState.round,
                 roundPricePerShare[depositReceipt.round],
-                vaultParams.decimals
+                vaultParams.decimals,
+                vaultParams.initialSharePrice
```

Comment:
Can remove this https://github.com/ribbon-finance/ribbon-v2/pull/38/files#r668738306